help/en/html/doc/Technical/gitadmin.shtml

```diff
@@ -110,7 +113,7 @@
         <li>Add any assignments beyond the original submitter.  
             The original submitter may
             be assigned automatically through a github action, but if not please add them.
-            Developers and maintainers can be assigned to any PR, but 
+            Developers, Approvers and Maintainers can be assigned to any PR, but 
```

Comment:
Should this say "Reviewers" instead of "Approvers"?  Or vice versa elsewhere?

Reply:
Reviewers sounds better than Approvers for me.

Reply:
Good catch - I ended up calling the team `Reviewers` as it does sound better than `Approvers` but missed have missed it when doing this. Will push an update.